addons/mail/static/src/models/thread_needaction_preview_view.js

```diff
@@ -3,6 +3,7 @@
 import { registerModel } from '@mail/model/model_core';
 import { attr, one } from '@mail/model/model_field';
 import { clear, insertAndReplace } from '@mail/model/model_field_command';
+import * as mailUtils from '@mail/js/utils';
```

Comment:
We generally prefer to explicitly list the functions we want to import, although I don't know if there is any particular reason for doing so.